src/resources/js/event-editor.js

```diff
@@ -3,21 +3,141 @@ var tribe_events_event_editor = tribe_events_event_editor || {};
 /**
  * Implements behaviours that are specific to the event editor.
  */
-jQuery( function( $ ) {
-	var obj = tribe_events_event_editor,
-		$sticky_in_month_view_checkbox = $( 'input[name="EventShowInCalendar"]' ),
-		$featured_event_checkbox = $( 'input[name="feature_event"]' );
+( function ( $, obj ) {
+	'use strict';
+
+	/**
+	 * Setup our selectors.
+	 *
+	 * @since TBD
+	 */
+	obj.selectors = {
+		sticky_in_month_view_checkbox: 'input[name="EventShowInCalendar"]',
+		featured_event_checkbox: 'input[name="feature_event"]',
+		organizer: {
+			area: '#event_tribe_organizer',
+			delete_button: '.tribe-delete-this',
+			add_button: '#event_tribe_organizer .tribe-add-post',
+			post_dropdown: '.linked-post-dropdown',
+			saved_organizers: '.saved-linked-post',
+		},
+	};
+
+	obj.organizer = {};
+
+	/**
+	 * Controls logic for the Organizer delete button to display.
+	 *
+	 * If more than one organizer exists, display the delete button.
+	 * If only one organizer exists, hide the delete button.
+	 *
+	 * @since TBD
+	 * @return void
+	 */
+	obj.organizer.deleteButtonDisplayLogic = () => {
```

Comment:
For consistency's sake, let's not use an arrow function here
```suggestion
	obj.organizer.deleteButtonDisplayLogic = function () {
```